src/states/node.rs

```diff
@@ -3012,6 +3019,11 @@ impl Node {
         self.peer_mgr.routing_table()
     }
 
+    /// Check whether the node acts ac tunnel_node for the tunnul pair.
```

Comment:
Couple of typos here.  Maybe "Check whether this node acts as a tunnel node between \`client_1\` and \`client_2\`"?  Whatever you choose, I'd duplicate it exactly for the corresponding comment in src/node.rs.